packages/orbit-components/src/BaggageStepper/Stepper/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for @kiwicom/orbit-components
+// Project: http://github.com/kiwicom/orbit
+
+import * as React from "react";
+
+import * as Common from "../../common/common";
+import { SharedProps, Event } from "../index";
+
+declare module "@kiwicom/orbit-components/lib/StepperStateless";
+
+type InputEvent = Common.Event<React.KeyboardEvent<HTMLInputElement>>;
+
+// TODO: fix types, onChange is not being used inside
```

Comment:
Why is it defined then?

Reply:
i think negative margin will not help, because of the shape of the buttons 🤔

Reply:
Will said, i can remove that gray bg of disabled input